internal/api/events_test.go

```diff
@@ -0,0 +1,57 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	kargoapi "github.com/akuity/kargo/api/v1alpha1"
+	"github.com/akuity/kargo/internal/server/user"
+)
+
+func TestFormatEventUserActorForOidc(t *testing.T) {
```

Comment:
Conventionally, we should use `Test<name of the function under test>`:

```suggestion
func TestFormatEventUserActor(t *testing.T) {
```